pkg/model/serializer.go

```diff
@@ -437,6 +444,34 @@ func (v ExternalVolume) MarshalYAML() (interface{}, error) {
 	return v.Name + ":" + v.SubPath + ":" + v.MountPath, nil
 }
 
+// UnmarshalYAML Implements the Unmarshaler interface of the yaml pkg.
+func (healthcheckProbes *HealthchecksProbes) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var rawBool bool
+	err := unmarshal(&rawBool)
+	if err == nil {
+		healthcheckProbes.Liveness = true
+		healthcheckProbes.Startup = true
+		healthcheckProbes.Readiness = true
+		return nil
+	}
+
+	var healthCheckProbesRaw healthCheckProbesRaw
+	err = unmarshal(&healthCheckProbesRaw)
+	if err != nil {
+		return err
+	}
+
+	healthcheckProbes.Liveness = healthCheckProbesRaw.Liveness
+	healthcheckProbes.Startup = healthCheckProbesRaw.Startup
+	healthcheckProbes.Readiness = healthCheckProbesRaw.Readiness
+	return nil
+}
+
+// MarshalYAML Implements the marshaler interface of the yaml pkg.
+func (healthcheckProbes HealthchecksProbes) MarshalYAML() (interface{}, error) {
+	return healthCheckProbesRaw(healthcheckProbes), nil
```

Comment:
if all are false, we shouldn't print anything
if all are true, a single `true` value should be returned
Also, can we have tests for these cases?